examples/userdemo/simple_annotate01.py

```diff
@@ -8,61 +8,38 @@
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
 
+
+fig, axs = plt.subplots(2, 4)
```

Comment:
Again, I don't mind it, but this one is pretty different from the old version.

Reply:
The previous example had a bunch of empty, useless subplots.
The reason for reordering the subplots is explained above, but again it's just so that "connectionstyle=arc3" doesn't get overwritten by the next subplot.